### ISSUE TITLE
Autoprefixer with Compass

### ISSUE THREAD

**Issue by drgullin** [bdkjones/CodeKit2#367]
I'm getting this issue once I add `require 'autoprefixer-rails'` to my `config.rb`:

```
Compass was unable to compile one or more files in the project: 

NoMethodError on line ["110"] of /Library/Ruby/Gems/2.0.0/gems/execjs-2.2.0/lib/execjs/external_runtime.rb: undefined method `split' for nil:NilClass
Run with --trace to see the full backtrace
```

That's what I've in `external_runtime.rb`, the reason of error is the `ENV['PATH']`

```
if File.executable? cmd
  cmd
else
  path = ENV['PATH'].split(File::PATH_SEPARATOR).find { |p|
    full_path = File.join(p, cmd)
    File.executable?(full_path) && File.file?(full_path)
  }
  path && File.expand_path(cmd, path)
end
```

I've also tried to use external paths for SASS and Compass, nothing changes.

Not sure why this happens, Prepros app works nice with a same code http://alphapixels.com/prepros/

**Comment by drgullin** [comment]
Any progress?

**Comment by bdkjones** [comment]
Not yet. Compass still doesn't let me control the exact output path of a file.

**Comment by drgullin** [comment]
This issue makes me sad, since I've to use an additional app for the compass and autoprefixer compilation (http://alphapixels.com/prepros/).

**Comment by bdkjones** [comment]
Implemented for 2.1.9.
